_posts/2020-01-16-bucket-loaders-released/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Google Cloud Loader 0.3.0 and S3 loader 0.7.0 released"
+title-short: Bucket loaders released
+tags: [snowplow, gcs, gcp, loader]
+author: Peel
+category: Releases
+permalink: /blog/2020/01/16/gcs-s3-loaders-released/
+---
+
+We are pleased to announce updated versions of our bucket loaders: [version 0.3.0][release-030] of Google Cloud Storage loader and [version 0.7.0][release-070] of S3 Loader that allow you to partition events by schema.
+
+<!--more-->
+
+<h2 id="partitioning">1. Partitioning by schema</h2>
+
+At Snowplow we use [self-describing-json][self-describing JSON] format to keep a well-defined, type-spec'd data definitions. When used with self-describing JSON, bucket loaders are now able to send each schema-formatted event to applicable schema directory in a tidy directory structure.
+
+The change comes along with R118 changes introducing a beta release of the new bad event format for easier post-processing.
```

Comment:
Let's add a link to the release post: https://snowplowanalytics.com/blog/2020/01/16/snowplow-release-r118-badrows/